pkg/util/mount/mount_linux.go

```diff
@@ -64,13 +66,13 @@ type Mounter struct {
 func (mounter *Mounter) Mount(source string, target string, fstype string, options []string) error {
 	bind, bindRemountOpts := isBind(options)
 	if bind {
-		err := doMount(mounter.mounterPath, source, target, fstype, []string{"bind"})
+		err := doMount(mounter.mounterPath, path.Join(mounter.mounterRootfsPath, source), target, fstype, []string{"bind"})
```

Comment:
`source` is already an absolute path in the native namespace. But from within the container, all native (root fs) paths have to be relative to the `mounterRootfsPath`.